templates/deployment.yaml

```diff
@@ -0,0 +1,68 @@
+---
+
+kind: Deployment
+apiVersion: extensions/v1beta1
+metadata:
+  name: {{ getSiteName() }}
+  namespace: {{ data.namespace }}
+spec:
+  replicas: {{ data.replicas | default(5) }}
+  template:
+    metadata:
+      labels:
+        app: {{ domain }}
+    spec:
+      containers:
+        - name: {{ getSiteName() }}
+          image: {{ data.image }}:{{ data.tag | default("latest", true) }}
+
+          ports:
+            - name: http
+              containerPort: {{ data.containerPort | default(80) }}
+
+          envFrom:
+            {%- if data.useProxy != False %}
```

Comment:
Actually, I just thought you might as well add all these environment variables directly to the `env:` block, but keep this logical wrapper.

Reply:
Done :)